### DISCUSSION TITLE
Possible to run on schedule?

### DISCUSSION THREAD

**Discussion by khthe8th** [louislam/dockge#392] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

Hi sorry I am new to docker and dockge and I am wondering if there is a way to schedule running one of my compose files? Specifically I have a plexmetamanager compose file I am running with the below compose, and it works great when I hit the start button in dockge.

Wondering if it's possible to automate pressing that start button every 24hrs?

version: "3.3"
services:
  plex-meta-manager:
    stdin_open: true
    tty: true
    volumes:
      - /volumes/pmm/config:/config:rw
    image: meisnate12/plex-meta-manager
    command: --run
networks: {}

### 🐻 Dockge Version

1.4.1

### 💻 Operating System and Arch

dockge lxc from tteck

### 🌐 Browser

firefox

**Comment by louislam** [comment, marked as answer]
I think you need to use cron. And this is an advantage of using Dockge, because you can control your stack by `docker-compose` commands

You can setup a cron job that run `cd /opt/stacks/your-stack && `docker-compose up -d` every day at maybe 00:00.

**Reply by khthe8th** [reply]
Sweet, thanks that worked perfectly!
